indexer/src/store.rs

```diff
@@ -369,6 +374,7 @@ impl DefaultIndexerStore {
                 })
         });
 
+        drop(sync_lock);
```

Comment:
lock will automatically unlock after leaving the scope, the `drop` is not necessary

Reply:
> lock will automatically unlock after leaving the scope, the `drop` is not necessary

I want to release lock asap, so calling the `drop` explicitly in all code blocks.